kale/embed/uncertainty_fitting.py

```diff
@@ -0,0 +1,121 @@
+import logging
```

Comment:
Could you add some docs at the top to explain what this module is about? More like a ReadMe for this API.